projects/observability/src/pages/explorer/explorer-dashboard-builder.ts

```diff
@@ -507,6 +559,19 @@ export class ExplorerDashboardBuilder {
               type: 'span-trace-navigation-handler'
             }
           },
+          {
+            type: 'table-widget-column',
```

Comment:
I'm a bit confused about this column because it was already added but I couldn't find where, there is no problem with having it two times anyways.